src/Resources/views/includes/layout/_content.html.twig

```diff
@@ -70,63 +70,6 @@
 
         {{ araise_table_render(content.table(view.data)) }}
 
-        {% set createUrl = content.createUrl(view.data) %}
-        {% if createUrl and (content.addVoterAttribute is null or is_granted(content.addVoterAttribute, view.data)) %}
-
-        <div
-            class="flex justify-end mt-3 mb-3 mx-3 {% if not isOnShow %}{{ content.option('attr')['class'] ?? '' }}{% endif %}"
-            {% if not isOnShow %}{{ attr|map((value, attr) => "#{attr}=\"#{value}\"")|join(' ')|raw }}{% endif %}
-            {{ stimulus_controller('araise/core-bundle/modal-form', { formUrl: createUrl }) }}
-        >
-            <button
-                type="button"
-                class="whatwedo-crud-button--action whatwedo-crud-button--small"
-                {{ stimulus_action('araise/core-bundle/modal-form', 'openModal') }}
-            >
-                {{ 'araise_crud.relation.add'|trans }}
-            </button>
-
-            <div
-                {{ stimulus_target('araise/core-bundle/modal-form', 'menu') }}
-                class="hidden whatwedo-crud-modal whatwedo-utility-modal"
-                data-transition-enter-from="opacity-0"
-                data-transition-enter-to="opacity-100"
-                data-transition-leave-from="opacity-100"
-                data-transition-leave-to="opacity-0"
-                role="dialog"
-                aria-modal="true"
-            >
-                <div class="whatwedo-crud-modal__overlay whatwedo-utility-modal__overlay">
-                    <div
-                        class="whatwedo-crud-modal__backdrop whatwedo-utility-modal__backdrop"
-                        {{ stimulus_action('araise/core-bundle/modal-form', 'close', 'click') }}
-                    ></div>
-
-                    {# This element is to trick the browser into centering the modal contents. #}
-                    <span class="hidden md:inline-block md:align-middle md:h-screen" aria-hidden="true">&#8203;</span>
```

Comment:
Nit: I still don't understand this hack. Centering stuff in CSS in 2024 works without "hacks". Do we still need this?

Reply:
There's somewhere a issue open by Mauri about that element. It's actually is needed the way solved it now. So just removing doesn't work (Mauri suggested that). So the element has it's purpose.

The fact that we're using that element is definitely not state of the art. It's the lightbox component from Tailwind UI.

Since we have some ideas about improve and changing the lightbox I don't want to leave it like that for now and adjust that automatically by removing Tailwind.

Reply:
@marcwieland95 

> There's somewhere a issue open by Mauri about that element. It's actually is needed the way solved it now. So just removing doesn't work (Mauri suggested that). So the element has it's purpose.

Then, once again, we can solve the confusion around this hack by adding a more detailed code remark about what this does. Reference it to the ticket or add more description that there is a known issue.

Questions around this hack will come over and over. The current comment does not address what the issue exactly is.